spec/channels/collect_submissions_channel_spec.rb

```diff
@@ -0,0 +1,43 @@
+describe CollectSubmissionsChannel, type: :channel do
+  context 'when a user can collect submissions' do
+    let!(:instructor) { create :instructor }
+    let!(:current_user) { User.find_by user_name: instructor.user_name }
```

Comment:
You can just do `instructor.user` here, and throughout (prefer association access over manual queries, at least for individual records)